### PR TITLE
Add inspector view

### PR DESCRIPTION
**Pull request by xepozz**
See https://github.com/yiisoft/yii-debug-api/pull/49

### CODE REVIEW

File: resources/views/inspector/view.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    <thead>
+    <tr>
+        <th>Test</th>
+        <th>Status</th>
+    </tr>
+    </thead>
+    <tbody></tbody>
+</table>
+
+<button id="button-load-params">Load params</button>
```
Comment (samdark) [inline resources/views/inspector/view.html:13]:
```suggestion
<button id="button-load-params">Parameters</button>
```

File: resources/views/inspector/view.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    <thead>
+    <tr>
+        <th>Parameter</th>
+        <th>Value</th>
+    </tr>
+    </thead>
+    <tbody></tbody>
+</table>
+
+<button id="button-load-config">Load config</button>
```
Comment (samdark) [inline resources/views/inspector/view.html:24]:
```suggestion
<button id="button-load-config">Configuration</button>
```

File: resources/views/inspector/view.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    <thead>
+    <tr>
+        <th>Parameter</th>
+        <th>Value</th>
+    </tr>
+    </thead>
+    <tbody></tbody>
+</table>
+
+<button id="button-load-container">Load container</button>
```
Comment (samdark) [inline resources/views/inspector/view.html:35]:
```suggestion
<button id="button-load-container">Container</button>
```

File: src/Panels/InspectorPanel.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    public function getName(): string
+    {
+        return 'Inspector';
+    }
+
+    public function renderSummary(): string
+    {
+        return '';
+        // TODO: hide it for now
+//        return file_get_contents(dirname(__DIR__, 2) . '/resources/views/inspector/summary.html');
```
Comment (samdark) [inline src/Panels/InspectorPanel.php:18]: ???

File: resources/views/inspector/view.html [hunk 5]
```diff
@@ -0,0 +1,142 @@
+<script src="https://unpkg.com/@alenaksu/json-viewer@0.3.4/dist/index.js"></script>
```
[the remark this answers is not in the document]
Comment (xepozz) [reply resources/views/inspector/view.html:1]: It's already used in other panels.

**Review by samdark — APPROVED** [review]

**Pull request merged**